emily/handler/tests/integration/deposit.rs

```diff
@@ -279,36 +321,46 @@ async fn get_deposits() {
 
     // Arrange.
     // --------
-    let bitcoin_txids: Vec<&str> = vec!["bitcoin_txid_1", "bitcoin_txid_2"];
-    let bitcoin_tx_output_indices = vec![1, 3, 2, 4]; // unordered.
+    let bitcoin_tx_output_indices: Vec<u32> = vec![0, 2, 1, 3]; // unordered.
 
+    let amounts = vec![DEPOSIT_AMOUNT_SATS; 4];
     // Setup test deposit transaction.
-    let DepositTxnData {
-        recipient: expected_recipient,
-        reclaim_script,
-        deposit_script,
-    } = DepositTxnData::new(DEPOSIT_LOCK_TIME, DEPOSIT_MAX_FEE, DEPOSIT_AMOUNT_SATS, 0);
+    let deposit_txn_data =
+        (0..2).map(|_| DepositTxnData::new(DEPOSIT_LOCK_TIME, DEPOSIT_MAX_FEE, &amounts));
```

Comment:
nit: this seems a bit convoluted, why not create this directly in the loop below?